how-to/workspace-platform-starter/client/src/framework/shapes/lifecycle-shapes.ts

```diff
@@ -101,3 +101,13 @@ export interface FavoriteChangedLifecyclePayload {
 	 */
 	favorite: FavoriteEntry;
 }
+
+/**
+ * Logged in event payload.
+ */
+export interface LoggedInLifecyclePayload {
+	/**
+	 * The user details.
+	 */
+	user: unknown;
```

Comment:
Make this optional? To indicate it might not be provided

Reply:
unknown include the undefined type, I guess it would make it more readable